packages/terra-file-path/package.json

```diff
@@ -31,7 +31,8 @@
     "classnames": "^2.2.5",
     "keycode-js": "^3.1.0",
     "prop-types": "^15.5.8",
-    "terra-collapsible-menu-view": "^6.86.0"
+    "terra-collapsible-menu-view": "^6.86.0",
+    "terra-visually-hidden-text": "2.37.0"
```

Comment:
Is visually hidden text component being used somewhere?

Reply:
Oh shoot good catch, I'll remove that now